### ISSUE TITLE
Add v2v commit hash to the UI

### ISSUE THREAD

**Issue by priley86** [ManageIQ/manageiq-v2v#422]
Per discussion w/ @AparnaKarve - i would like to look into adding the last v2v commit hash somewhere into the MIQ UI. I think it would probably be easiest to place this in the existing "About Modal" like we do with the MIQ commit hash and add a section for "V2V Version".

![screen shot 2018-06-20 at 8 43 41 am](https://user-images.githubusercontent.com/4237045/41659112-603349f4-7466-11e8-95b7-7009073d18fb.png)

thoughts @vconzola ?

Otherwise, we can probably obscure this version and write it out somewhere in the HTML or in the Console logs...

**Comment by AllenBW** [comment]
OH I LIKE THIS IDEA!!! 
maybe something like... 
<img width="604" alt="screen shot 2018-06-20 at 8 55 52 am" src="https://user-images.githubusercontent.com/6640236/41659625-cbe1184c-7467-11e8-96fc-dac606392925.png">

**Comment by vconzola** [comment]
@priley86 This looks good to me. Let's bring it up in the sync meeting later today.

**Comment by priley86** [comment]
Yes - exactly @AllenBW

**Comment by AparnaKarve** [comment]
Since the About box is a ui-classic component, we would have to review this idea with some of the ui-classic folks first.
@priley86 In case if adding the hash in the  About box box does not work out for whatever reason, could we stick the hash somewhere in the Migration page html, like you originally suggested?

**Comment by AllenBW** [comment]
Good news everyone!!

I was talking to @Fryguy and he said... (👈 this is the joke part)

TL;DR - If we're gonna list one plugin, we have to list em all. About Modal is fine, but we might want to have a scrolling boxy thing.  

### This is what "em all" really means
(from rails console `Vmdb::Plugins.instance.vmdb_plugins.map { |p| p.class.name }` produces this)

```
["ManageIQ::Providers::AnsibleTower::Engine",
 "ManageIQ::Providers::Amazon::Engine",
 "ManageIQ::Providers::Azure::Engine",
 "ManageIQ::Providers::Foreman::Engine",
 "ManageIQ::Providers::Google::Engine",
 "ManageIQ::Providers::Kubernetes::Engine",
 "ManageIQ::Providers::Kubevirt::Engine",
 "ManageIQ::Providers::Lenovo::Engine",
 "ManageIQ::Providers::Nuage::Engine",
 "ManageIQ::Providers::Redfish::Engine",
 "ManageIQ::Providers::Openshift::Engine",
 "ManageIQ::Providers::Openstack::Engine",
 "ManageIQ::Providers::Ovirt::Engine",
 "ManageIQ::Providers::Scvmm::Engine",
 "ManageIQ::Providers::Vmware::Engine",
 "ManageIQ::AutomationEngine::Engine",
 "ManageIQ::Api::Engine",
 "ManageIQ::GraphQL::Engine",
 "ManageIQ::Content::Engine",
 "ManageIQ::Showback::Engine",
 "ManageIQ::V2V::Engine",
 "ManageIQ::UI::Classic::Engine"]
```
And we gotta list the hash of all of em, which can be gotten from a number of places, gem.lock, `git log -1 --oneline`, bundler...  The list goes on...  There's this other complication being the difference in commits upstream vs downstream, the issue being that a commit made downstream will not have a hash that maps to anything upstream, rendering the info uselessssssssss.  So what 👈 means, we gotta be real smart about from where we pull the hash. 

All n All, this is a great idea, for QE, CVC, customers (but maybe less so) an easy way to access the versions of all the pieces that go into CF would be great for troubleshooting.

@vconzola imma see if Jason can pop into the api sync to talk a lil bit about this...

**Comment by priley86** [comment]
thanks again for researching this @AllenBW 

Filed a BZ for this for future sprints:
https://bugzilla.redhat.com/show_bug.cgi?id=1594348

**Comment by Fryguy** [comment]
@AllenBW This should not be closed...#436 only implemented part of the proposal, and did so in a way that didn't handle any of the caveats mentioned in https://github.com/ManageIQ/miq_v2v_ui_plugin/issues/422#issuecomment-398808617

**Comment by AllenBW** [comment]
@Fryguy githooks at work 😏

**Comment by mturley** [comment]
Looks like all plugins are listed in the about modal now. Closing.
